src/mark.ts

```diff
@@ -678,7 +690,10 @@ export interface TickConfig<ES extends ExprRef | SignalRef> extends MarkConfig<E
 }
 
 export const defaultTickConfig: TickConfig<SignalRef> = {
-  thickness: 1
+  thickness: 1,
+  discreteBandSize: {band: 0.75},
```

Comment:
<img width="568" alt="image" src="https://github.com/vega/vega-lite/assets/111269/c749d5e3-5d6c-4782-bcb7-5a45c58d605d">

0.75 here are mainly to match 3/4 multiplier in the old code: 

<img width="568" alt="image" src="https://github.com/vega/vega-lite/assets/111269/150cfc88-6e41-4282-845a-472651d5cc72">

Reply:
The two screenshots look the same to me.